packages/css_colors/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
```

Comment:
This needs a version change, which is why CI is failing.

Reply:
Updated